### PR TITLE
Updated example 

### PR DESCRIPTION
**Pull request by jmatsushita**
This should deal with #20 i.e. make the example work with psc 0.10

### CODE REVIEW

File: examples/central-counter/frontend/bower.json [hunk 1]
```diff
@@ -8,63 +8,70 @@
...
-    "purescript-aff": "^1.0.0",
-    "purescript-argonaut-generic-codecs": "^3.0.0",
-    "purescript-servant-support": "^5.0.0",
-    "purescript-subscriber": "^1.0.0",
+    "purescript-pux": "^7.0.0",
+    "purescript-affjax": "^3.0.0",
+    "purescript-aff": "^2.0.0",
+    "purescript-argonaut-generic-codecs": "^4.0.0",
+    "purescript-servant-support": "^6.0.0",
+    "purescript-subscriber": "https://github.com/iilab/purescript-subscriber.git#master",
```
Comment (jmatsushita) [inline examples/central-counter/frontend/bower.json:16]: That's temporary until this https://github.com/eskimor/purescript-subscriber/pull/7 is merged and published.

Comment (eskimor) [reply examples/central-counter/frontend/bower.json:16]: Merged and published! Thank you!

**Comment by eskimor** [comment]
Looks very good! :-) Thank you!

**Comment by eskimor** [comment]
@jmatsushita could you fix purescript-subscriber dep? Then I can merge this PR. Thank you!

**Comment by eskimor** [comment]
I can't test, but I assume I'll get that fix right by myself ;-)

**Pull request merged**